.storybook/preview.tsx

```diff
@@ -15,13 +16,30 @@ export const parameters = {
   },
 };
 
+const queryClient = new QueryClient();
+
 addDecorator((Story) => (
-  <WalletProvider>
-    <QueryClientProvider client={new QueryClient()}>
-      <PanelProvider>
-        <GlobalStyle />
-        <Story />
-      </PanelProvider>
-    </QueryClientProvider>
-  </WalletProvider>
+  <ErrorProvider>
+      <VoteTimingProvider>
+        <WalletProvider>
```

Comment:
Updated the storybook preview with the new contexts that have been added recently. The preview should always mirror the `_app.tsx` file